python/ray/dag/compiled_dag_node.py

```diff
@@ -63,13 +63,16 @@ def do_allocate_channel(
     self,
     reader_and_node_list: List[Tuple["ray.actor.ActorHandle", str]],
     typ: ChannelOutputType,
+    num_shm_buffers: int,
 ) -> ChannelInterface:
     """Generic actor method to allocate an output channel.
 
     Args:
         reader_and_node_list: A list of tuples, where each tuple contains a reader
             actor handle and the node ID where the actor is located.
         typ: The output type hint for the channel.
+        num_shm_buffers: The number of shared memory buffer per channel.
```

Comment:
How about calling it `num_channel_buffers`? The number of buffers a channel has.
It does not talk about impl details of shared memory

Reply:
changed it to max_in_flight_requests